klc-cert-manager/config/manager/manager.yaml

```
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: klc-cert-manager
```

Please, do a preg replace all 😅 
```suggestion
    control-plane: klt-cert-manager
```